sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -753,10 +753,6 @@ def test_check_estimator():
     msg = "object has no attribute 'fit'"
     with raises(AttributeError, match=msg):
         check_estimator(BaseEstimator())
-    # check that fit does input validation
-    msg = "Did not raise"
```

Comment:
now that we assert the right message, it became clear that this was already tested in another test and was redundant.